### PR TITLE
Portfolio details popup window

### PR DESCRIPTION
**Pull request by LethaboMot**
in the project:

- added the CSS for the styling of the models
- added more javascript for the page to be more dynamic
- made the mobile version of the popup
- made the desktop version of the popup

### CODE REVIEW

File: modal.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+span3.onclick = function () {
+  popup3.style.display = 'none';
+};
+
+window.onclick = function (event) {
+  if (event.target === popup3) {
+    popup3.style.display = 'none';
+  }
+};
```
Comment (Ibizugbe) [inline modal.js:77]: Kindly ensure you keep your code DRY as this is a requirement for our [JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) best practices😉 Let's avoid repetition here by making use of a `loop`. you already created an `array` we can loop through its content and use only one of the function call on the click of each button👍

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @LethaboMot 👋👋, 

tryagain


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅

- [x] Linrters are properly setup and they are passing👍.
- [x] When the user clicks (or taps) the button to check project details, the popup with details about the project appears👍.
- [x] When the user clicks (or taps) the close (X) button, the popup disappears👍.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] [OPTIONAL] Kindly note that your entire project is looking a bit different from the [given design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34-848&t=nSrU7uvc0RxW1agh-0) kindly take a look at the previous milestone tasks as regards design and take make comparison and make necessary changes as it is a display of professionalism when one always sticks to the given design😉.
- [ ] [OPTIONAL] Nice work on your commit messages as they are descriptive😉 but we can improve them by ensuring they are always capitalised😉 you can take a look at[ this lesson](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for more clarity😉

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @LethaboMot          👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights
- Proper use of javascript to render popup dynamically ✔️ 
- Moderate UI styling 👍 
  
# Required Changes ♻️

- There's a bit of a syntax issue  that needs to be an adjustment based on best practices 

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: modal.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+const popup = document.getElementById('myPopup');
```
Comment (huseyinbiyik) [inline modal.js:1]: Unfortunately, your first See Project button opens the popup but other projects do not. Also, icons on the popup window look broken. These two problems were not there before the last review. They introduced after that 😞  Kindly fix the issues and make sure they work as before with your new refactored code 👍 
lala

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- .gitignore used to ignore requested files 📁
- PR description is descriptive 📝


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shisui6 — APPROVED** [review]
Hi @LethaboMot, 

# Approved 🪄
Your project is complete! There is nothing else to say other than... it's time to merge it 💎
Congratulations! 🎉 

## Highlights
- Popup window implemented ✅


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer, and addressing them will really improve your application._

You can also consider the following:

N/A

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**